src/Controls/src/Core/Button/Button.iOS.cs

```diff
@@ -409,7 +409,7 @@ static bool ResizeImageIfNecessary(UIButton platformButton, Button button, UIIma
 					return false;
 				}
 
-				image = image?.ImageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal);
+				image = image?.ImageWithRenderingMode(UIImageRenderingMode.Automatic);
```

Comment:
The test `ImageSourceInitializesCorrectly` from the Issue24916 class is failing on iOS.

The generated result with the changes is:
![image](https://github.com/user-attachments/assets/7df82556-10da-40ab-ae7a-ea41138f0cb8)

While the code is:

```
<VerticalStackLayout>
        <Label Text="Buttons:" AutomationId="label"/>
        <Button ImageSource="red.png"/>
        <Button ImageSource="green.png"/>
        <Button ImageSource="blue.png"/>
        <Label Text="Image buttons:"/>
        <ImageButton Source="red.png"/>
        <ImageButton Source="green.png"/>
        <ImageButton Source="blue.png"/>
 </VerticalStackLayout>
```

https://github.com/dotnet/maui/blob/8ac1263c19a3aefc83f56474cbce942bdcae127a/src/Controls/tests/TestCases.HostApp/Issues/Issue24916.xaml#L26

Should render 3 buttons, red, green and blue with that order. With the changes, all al rendered with the latest image.

Reply:
> The test `ImageSourceInitializesCorrectly` from the Issue24916 class is failing on iOS.
> 
> The generated result with the changes is: ![image](https://private-user-images.githubusercontent.com/6755973/374586960-7df82556-10da-40ab-ae7a-ea41138f0cb8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2MzU5OTUsIm5iZiI6MTcyODYzNTY5NSwicGF0aCI6Ii82NzU1OTczLzM3NDU4Njk2MC03ZGY4MjU1Ni0xMGRhLTQwYWItYWU3YS1lYTQxMTM4ZjBjYjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTFUMDgzNDU1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDQ4NmQyODllMDc0NzE4ODRlYmI3MGE4MDU0NDMxODA2NmQ4Mjg4ODdjYzIxNzUyNTA0M2UwY2QwMjcxMTUzYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.eESiqj8CX-IWpo1bcMPbNTEmm9C1dbQVz55FB43fPjo)
> 
> While the code is:
> 
> ```
> <VerticalStackLayout>
>         <Label Text="Buttons:" AutomationId="label"/>
>         <Button ImageSource="red.png"/>
>         <Button ImageSource="green.png"/>
>         <Button ImageSource="blue.png"/>
>         <Label Text="Image buttons:"/>
>         <ImageButton Source="red.png"/>
>         <ImageButton Source="green.png"/>
>         <ImageButton Source="blue.png"/>
>  </VerticalStackLayout>
> ```
> 
> https://github.com/dotnet/maui/blob/8ac1263c19a3aefc83f56474cbce942bdcae127a/src/Controls/tests/TestCases.HostApp/Issues/Issue24916.xaml#L26
> 
> Should render 3 buttons, red, green and blue with that order. With the changes, all al rendered with the latest image.

It is working now and allows devs for the tint color modifications :)